modules/nginx/templates/default-vhost.conf

```diff
@@ -1,5 +1,13 @@
 server {
   listen 80 default_server;
+
+  <% if scope.lookupvar('::aws_migration') %>
```

Comment:
Actually I think it's going to add a white line that will refresh all Nginx processes